src/OpenTelemetry.Instrumentation.AspNetCore/AspNetCoreTraceInstrumentationOptions.cs

```diff
@@ -24,6 +28,26 @@ namespace OpenTelemetry.Instrumentation.AspNetCore;
 /// </summary>
 public class AspNetCoreTraceInstrumentationOptions
 {
+    /// <summary>
+    /// Initializes a new instance of the <see cref="AspNetCoreTraceInstrumentationOptions"/> class.
+    /// </summary>
+    public AspNetCoreTraceInstrumentationOptions()
+        : this(new ConfigurationBuilder().AddEnvironmentVariables().Build())
+    {
+    }
+
+    internal AspNetCoreTraceInstrumentationOptions(IConfiguration configuration)
+    {
+        Debug.Assert(configuration != null, "configuration was null");
+
+#if NET6_0_OR_GREATER
+        if (configuration.TryGetBoolValue("OTEL_DOTNET_EXPERIMENTAL_ENABLE_GRPC_INSTRUMENTATION", out var enableGrpcInstrumentation))
```

Comment:
Does this only have support for Environment Variables?

Reply:
Changelog and readme should be showing that this can be configured not just from env variables.